src/main/java/it/gov/pagopa/reward/notification/controller/NotificationController.java

```diff
@@ -45,6 +46,34 @@ Mono<Page<RewardExportsDTO>> getExportsPaged(
             @PageableDefault(size = 10) Pageable pageable,
             ExportFilter filters);
 
+    @GetMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/reward/notification/exports/{exportId}/summary")
+    Mono<ExportSummaryDTO> getSingleExportSummary(
+            @PathVariable("organizationId") String organizationId,
+            @PathVariable("initiativeId") String initiativeId,
+            @PathVariable("exportId") String exportId);
+
+    @GetMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/reward/notification/exports/{exportId}")
+    Flux<ExportDetailDTO> getSingleExport(
+            @PathVariable("organizationId") String organizationId,
+            @PathVariable("initiativeId") String initiativeId,
+            @PathVariable("exportId") String exportId,
+            @PageableDefault(size = 10) Pageable pageable,
+            ExportDetailFilter filters);
+
+    @GetMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/reward/notification/exports/{exportId}/paged")
+    Mono<ExportPageDTO> getSingleExportPaged(
+            @PathVariable("organizationId") String organizationId,
+            @PathVariable("initiativeId") String initiativeId,
+            @PathVariable("exportId") String exportId,
+            @PageableDefault(size = 10) Pageable pageable,
+            ExportDetailFilter filters);
+
+    @GetMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/reward/notification/exports/event/{eventId}")
```

Comment:
why "event" and "eventId"? we are talking about rewardNotifications, thus notificationId or rewardNotificationId :)
```suggestion
    @GetMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/reward/notification/{notificationId}")
```